pyro/infer/mcmc/api.py

```diff
@@ -412,7 +412,8 @@ def model(data):
             elif self.kernel.model:
                 _, _, self.transforms, _ = initialize_model(self.kernel.model,
                                                             model_args=args,
-                                                            model_kwargs=kwargs)
+                                                            model_kwargs=kwargs,
+                                                            initial_params={})
```

Comment:
Do you want to default this to `None` rather than `{}`?

Reply:
If we set this to None, it will try to find valid initial params.

Reply:
Thanks for explaining!